src/Microsoft.AspNet.Mvc.Core/ActionResults/ContentResult.cs

```diff
@@ -26,17 +29,34 @@ public class ContentResult : ActionResult
         {
             var response = context.HttpContext.Response;
 
+            // Encoding property on MediaTypeHeaderValue does not return the exact encoding instance that
```

Comment:
We shouldn't need to this. If using the encoding set in `ContentType` is incorrect, we have an issue with the way `MediaTypeHeaderValue.Encoding` behaves. cc @Tratcher